package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@boutinflegel/sw-rpg-icons",
```

Comment:
package should likely be renamed to ffg icons or separate packages for the other ips